analyzers/tests/SonarAnalyzer.Test/TestCases/EmptyMethod.CSharp9.Comment.Fixed.cs

```diff
@@ -133,3 +125,59 @@ partial class C
         }
     }
 }
+
+class PropertyAccessors
+{
+    int NonEmptyInitProp { init { int x; } }
+    int EmptyInitProp {
+        init
+        {
+            // Method intentionally left empty.
+        }
+    }                   // Fixed
+    int EmptyInitPropWithGet { get => 42; init
+        {
+            // Method intentionally left empty.
+        }
+    } // Fixed
+    int AutoInitPropWithGet { get; init; }           // Compliant, auto-implemented, so not-empty
+
+    int NonEmptySetProp { set { int x; } }
+    int EmptySetProp {
+        set
+        {
+            // Method intentionally left empty.
+        }
+    }                     // Fixed
+    int EmptySetPropWithGet { get => 42; set
+        {
+            // Method intentionally left empty.
+        }
+    }   // Fixed
+    int AutoSetPropWithGet { get; set; }             // Compliant, auto-implemented, so not-empty
+
+    class Base
+    {
+        protected virtual int VirtualEmptyInitProp { init { } }  // Compliant, virtual
+    }
+
+    class Inherited : Base
+    {
+        protected override int VirtualEmptyInitProp {
+            init
```

Comment:
Note for the reviewer: in the presence of accessors with empty bodies, I have decided to go only one level up w.r.t. the empty body, when formatting the code fix, instead of formatting at the property level.
That is because the user may also have other accessors defined in the property, that are not involved in the code fix. 

In that scenario, a code fix on the body of an accessor would end up formatting a potentially large portion of user code, that may not be what the user wants.

```cs
int AProperty
{
    get { /* ... */ } // Get accessor with code, that the user may not want to be reformatted
    init { } 
}
```

Let me know if you think that it would be better, in your opinion, to take the risk of reformatting unrelated user code, and have better formatting overall, and I will make the change.